tests/execution/apollo-federation-entities.md

```diff
@@ -0,0 +1,74 @@
+# Apollo federation query
+
+```graphql @config
+schema
+  @server(port: 8000)
+  @upstream(baseURL: "http://jsonplaceholder.typicode.com", httpCache: 42, batch: {delay: 100}) {
+  query: Query
+}
+
+type Query {
+  user(id: Int!): User @http(path: "/users/{{.args.id}}")
+}
+
+type User @call(steps: [{query: "user", args: {id: "{{.value.user.id}}"}}]) {
```

Comment:
TODO: Check if `n+1` is reported on the CLI on this query path.

Reply:
Let's track it in a different issue.

Reply:
https://github.com/tailcallhq/tailcall/issues/2892